config/application.rb

```diff
@@ -78,10 +78,11 @@ class Application < Rails::Application
     config.logger = config.lograge.logger
     config.log_level = ENV["LOG_LEVEL"].to_sym
 
-    config.lograge.ignore_actions = ["HeartbeatController#index", "IndexController#index"]
-    config.lograge.ignore_custom = lambda do |event|
-      event.payload.inspect.length > 100000
-    end
+    # config.lograge.ignore_actions = ["HeartbeatController#index", "IndexController#index"]
+    # config.lograge.ignore_custom = lambda do |event|
+    #   event.payload.inspect.length > 100000
+    # end
+
```

Comment:
Do we need this config in the latest version? is this version specific configuration?